src/pages/QuizUi/QuestionContent.jsx

```diff
@@ -109,10 +116,27 @@ export default function QuestionContent({
             'radio',
             selected,
             setSelected,
-            isCurrentQuestionAnswered
+            isCurrentQuestionAnswered,
+            correctnessInfo,
+            setSelectedOption
           )}
         </RadioGroup>
       )}
+      {isCurrentQuestionAnswered &&
+        correctnessInfo &&
+        (correctnessInfo.isCorrect ? (
+          <Typography> Nice work! </Typography>
+        ) : (
+          <>
+            <Typography> Sorry, that is incorrect. </Typography>
+            {resources && (
+              <Typography>
+                You can learn more about this topic at the following links:
```

Comment:
This is a great feature! Are there any quizzes with resources? I think it'd be valuable for the user to see this whether or not they get the answer correct.

Reply:
There is a resource key value! we just need to add the actual resources, which we hopefully will get to!